index.js

```diff
@@ -314,335 +385,85 @@ exports.main = function() {
-  setupTicker('KrakenDogecoin', 'Kraken', 'DOGE', '\u0243', '#000066', "https://api.kraken.com/0/public/Ticker?pair=XBTXDG", ['result','XXBTXXDG','c',0]);
-
-  // Namecoin prices //
-  setupTicker('BTCeNamecoinUSD', 'BTCe', '$', 'NMC', '#413ADF', "https://btc-e.com/api/2/nmc_usd/ticker", ['ticker','last']);
-
-  // Auroracoin prices //
-  setupTicker('CryptsyAuroracoin', 'Cryptsy', '\u0243', 'AUR', '#413ADF', "http://pubapi.cryptsy.com/api.php?method=singlemarketdata&marketid=160", ['return','markets','AUR','lasttradeprice']);
-
-  // Blackcoin prices //
-  setupTicker('CryptsyBlackcoin', 'Cryptsy', '\u0243', 'BC', '#000', "http://pubapi.cryptsy.com/api.php?method=singlemarketdata&marketid=179", ['return','markets','BC','lasttradeprice']);
-
-  // Nxt prices //
-  setupTicker('CryptsyNxt', 'Cryptsy', '\u0243', 'NXT', '#000', "http://pubapi.cryptsy.com/api.php?method=singlemarketdata&marketid=159", ['return','markets','NXT','lasttradeprice']);
-  setupTicker('PoloniexNxt', 'Poloniex', '\u0243', 'NXT', '#000', "https://poloniex.com/public?command=returnTicker", ['BTC_NXT', 'last']);
-
-  // Bitshares prices //
-  setupTicker('PoloniexBitshares', 'Poloniex', '\u0243', 'BTS', '#000', "https://poloniex.com/public?command=returnTicker", ['BTC_BTS', 'last']);
-
-  // Ripple prices //
-  setupTicker('KrakenRipple', 'Kraken', 'XRP', '\u0243', '#000066', "https://api.kraken.com/0/public/Ticker?pair=XBTXRP", ['result','XXBTXXRP','c',0]);
-
-  // Maidsafe prices //
-  setupTicker('PoloniexMaidsafe', 'Poloniex', '\u0243', 'MAID', '#000', "https://poloniex.com/public?command=returnTicker", ['BTC_MAID', 'last']);
-
-  // BitcoinDark prices //
-  setupTicker('PoloniexBitcoindark', 'Poloniex', '\u0243', 'BTCD', '#000', "https://poloniex.com/public?command=returnTicker", ['BTC_BTCD', 'last']);
-
-  // Monero prices //
-  setupTicker('PoloniexMonero', 'Poloniex', '\u0243', 'XMR', '#000', "https://poloniex.com/public?command=returnTicker", ['BTC_XMR', 'last']);
-
-  // Dash prices //
-  setupTicker('CryptsyDashBTC', 'Cryptsy', '\u0243', 'DASH', '#000', "http://pubapi.cryptsy.com/api.php?method=singlemarketdata&marketid=155", ['return','markets','DRK','lasttradeprice']);
-  setupTicker('BitFinexDashBTC', 'BitFinex', '\u0243', 'DASH', '#000', "https://api.bitfinex.com/v1/ticker/drkbtc", ['last_price']);
-  setupTicker('CryptsyDashUSD', 'Cryptsy', '$', 'DASH', '#000', "http://pubapi.cryptsy.com/api.php?method=singlemarketdata&marketid=213", ['return','markets','DRK','lasttradeprice']);
-  setupTicker('BitFinexDashUSD', 'BitFinex', '$', 'DASH', '#000', "https://api.bitfinex.com/v1/ticker/drkusd", ['last_price']);
-
-  // Burst prices //
-  setupTicker('PoloniexBurst', 'Poloniex', '\u0243', 'BURST', '#fff', "https://poloniex.com/public?command=returnTicker", ['BTC_BURST', 'last']);
-
-
-  loadTickersInOrder(); // Load and create all selected tickers
+  function registerEvents() {
+    for (tickerId in tickers) {
```

Comment:
Creating global 'for' variable. Should be 'for (var tickerId ...'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.<br>Too many errors. (95% scanned).